react-props/my-app/src/Button.tsx

```diff
@@ -0,0 +1,7 @@
+type Button = {
```

Comment:
You shouldn't have both a type and a function with the same name because it gets confusing. I prefer to name this `Props` everywhere since that's pretty simple. You could also use `ButtonProps` or something.